docs/server/source/appendices/commands.rst

```diff
@@ -6,10 +6,10 @@ Command Line Interface
     :special-members: __init__
 
 
-:mod:`bigchaindb.commands.bigchain`
+:mod:`bigchaindb.commands.bigchaindb`
 -----------------------------------
```

Comment:
This doesn't seem like to be the issue. I added the two characters and am still getting this error when running `make docs`.

> 
> > Extension error:
> Could not import extension generate_schema_documentation (exception: No module named 'bigchaindb')
> Makefile:56: recipe for target 'html' failed
> make[1]: *** [html] Error 1
> make[1]: Leaving directory '/home/lavina/bigchaindb/docs/server'
> Makefile:68: recipe for target 'docs' failed
> make: *** [docs] Error 2